vendor/orchestra-types-rs/src/proto/messages.proto

```diff
@@ -0,0 +1,96 @@
+syntax = "proto3";
+
+package orchestra.messages;
+
+// The ProtocolStateStreamer service definition.
+service ChainStateStreamer {
+    // Get Bitcoin chain tip
+    rpc GetBitcoinChainTip(None) returns (BitcoinBlockData);
+    // Get Stacks chain tip
+    rpc GetStacksChainTip(None) returns (BitcoinBlockData);
```

Comment:
Should `BitcoinBlockData` be renamed if it is used for both Bitcoin and Stacks blocks?

Reply:
Addressed with https://github.com/hirosystems/clarinet/pull/402/commits/59633832515a2974261eb8e5886e9632d62695b6